### ISSUE TITLE
Support for Hyperlinks

### ISSUE THREAD

**Issue by crhuber** [cooklang/cooklang-ios-app#21]
Sometimes in recipes I add a url to the original page where I found it. Sometimes to check instructional video or photos. 
Would be nice to have them clickable

**Comment by dubadub** [comment]
@crhuber great, thanks for suggestion. Did I understand you correctly: you propose to display a url from the source metadata on a recipe screen, right?

**Comment by crhuber** [comment]
Yes in the source metadata. If present would open in an external browser
